velox/dwio/parquet/thrift/ParquetThriftTypes.cpp

```diff
@@ -1038,13 +1036,12 @@ void swap(MapType& a, MapType& b) {
   (void)b;
 }
 
-MapType::MapType(const MapType& other6) {
-  (void)other6;
-}
-MapType& MapType::operator=(const MapType& other7) {
-  (void)other7;
+MapType::MapType(const MapType& /*other6*/) {}
```

Comment:
@czentgr : I'm unsure how those swap calls got missed. It seems rather odd to do nothing for a swap, but since its generated code I presume it is optimized to do nothing when the types are not parameterized.

I have cleared out the swap calls in that case now.

Reply:
Maybe the symbols need to be present for these types, so maybe template generation works out when these functions are used in generic code.
We would get build/runtime errors if something is missing as a result of optimization.